src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -85,10 +85,9 @@ function update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
         )
         @. S_sfc = ∫H_S_e_dz / ∫H_dz
     else
+        @. p.soil.sub_sfc_scratch = effective_saturation(ν, p.soil.θ_l, θ_r)
```

Comment:
this removes an allocation too, but I dont think this branch is being used with our current config